src/components/MuiVirtualizedTable/ColumnHeader.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import clsx from 'clsx';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme) => ({
+    label: {
+        fontWeight: 'bold',
+        padding: '16px',
+    },
+    divFlex: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+    },
+    divNum: {
+        display: 'flex',
+        flexDirection: 'row-reverse',
+        alignItems: 'center',
+    },
+    sortDiv: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    sortButton: {
+        width: '1em',
+        height: '2em',
+        fill: 'currentcolor',
+    },
+    filterButton: {
+        width: '1em',
+        height: '1em',
+        fill: 'none',
+        stroke: 'currentcolor',
+        strokeWidth: '3',
+    },
+    tr180: {
+        transform: 'rotate(180deg)',
+    },
+    filterTooLossy: {
+        fill: theme.palette.secondary.main,
+    },
+    transparent: {
+        opacity: 0,
+    },
+    hovered: {
+        opacity: 0.5,
+    },
+}));
+
+const SortButton = (props) => {
+    const classes = useStyles();
+    const sortRank = Math.abs(props.signedRank);
+    const visibilityClass =
+        !props.signedRank &&
+        (props.headerHovered ? classes.hovered : classes.transparent);
+    return (
+        <>
+            <div className={clsx(classes.sortDiv)} onClick={props.onClick}>
+                <svg
```

Comment:
Why not using IconButton Component from MUI and `'@mui/icons-material/SortOutlined'`

Reply:
Done